specification/sections/01-flag-evaluation.md

```diff
@@ -33,6 +35,12 @@ See [provider](./02-providers.md) for details.
 
 #### Requirement 1.1.3
 
+> The `provider mutator` function **MUST** run the `initialize` function on the newly registered provider.
```

Comment:
It would be an internal example. I've thus far generally avoided example implementations of internal SDK functions.

I did recently add more to the linked provider initialization example.